src/DataValues/QuantityValue.php

```diff
@@ -91,155 +70,59 @@ public function __construct( DecimalValue $amount, $unit, DecimalValue $upperBou
 	 * @note: if the amount or a bound is given as a string, the string must conform
 	 * to the rules defined by @see DecimalValue.
 	 *
-	 * @since 0.1
+	 * @since 0.7
 	 *
-	 * @param string|int|float|DecimalValue $amount
+	 * @param string|int|float|DecimalValue|UnboundedQuantityValue $number
```

Comment:
I believe this is a mistake. We talked about this and decided to not allow using an other QuantityValue as a number because you can pass unit and bounds twice that way.

Same a few lines below.